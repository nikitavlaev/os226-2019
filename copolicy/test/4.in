fifo
rtapp 1 -1 10
rtapp 2 -1 10
