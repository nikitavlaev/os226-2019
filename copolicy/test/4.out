rtapp id 0 cnt 9
rtapp id 1 cnt 9
rtapp id 0 cnt 8
rtapp id 1 cnt 8
rtapp id 0 cnt 7
rtapp id 1 cnt 7
rtapp id 0 cnt 6
rtapp id 1 cnt 6
rtapp id 0 cnt 5
rtapp id 1 cnt 5
rtapp id 0 cnt 4
rtapp id 1 cnt 4
rtapp id 0 cnt 3
rtapp id 1 cnt 3
rtapp id 0 cnt 2
rtapp id 1 cnt 2
rtapp id 0 cnt 1
rtapp id 1 cnt 1
rtapp id 0 cnt 0
rtapp id 1 cnt 0
